dask/dataframe/tests/test_dataframe.py

```diff
@@ -762,21 +762,17 @@ def test_dropna():
     )
 
     # threshold
-    assert_eq(df.dropna(thresh=None), df.loc[[20, 40]])
```

Comment:
I removed a few asserts which were testing `pandas` but not `dask` (note these are using `df` instead of `ddf`). Not sure why these were added originally, but seem out of scope for our test suite.